code/modules/mob/living/living.dm

```diff
@@ -367,6 +367,7 @@
 /mob/living/stop_pulling()
 	if(ismob(pulling))
 		reset_pull_offsets(pulling)
+		log_game("[key_name(src)] stopped pulling [pulling] in [AREACOORD(src)]")
```

Comment:
```suggestion
		src.log_message("stopped pulling [pulling] in [AREACOORD(src)]", LOG_ATTACK)
```

This should probably be in attack log as grabs & pulled from are already located there. Also use log_message to ensure admins get it on mob/user logs in-game